projects/system-semconv.md

```diff
@@ -0,0 +1,86 @@
+
+## Project Board
+
+See: <https://github.com/orgs/open-telemetry/projects/55/views/3>
+
+### Roadmap
+
+- <https://github.com/open-telemetry/semantic-conventions/issues/127>
+- ([@ChrsMark]) <https://github.com/open-telemetry/semantic-conventions/issues/130>
+- ([@mx-psi](https://github.com/mx-psi)) <https://github.com/open-telemetry/semantic-conventions/issues/131>
+- ([@frzifus](https://github.com/frzifus)) <https://github.com/open-telemetry/semantic-conventions/issues/73>
+- ([@dineshg13](https://github.com/dineshg13)) <https://github.com/open-telemetry/semantic-conventions/issues/647>
+- <https://github.com/open-telemetry/semantic-conventions/issues/649>
+- ([@braydonk](https://github.com/braydonk)) <https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/7417>
+- <https://github.com/open-telemetry/semantic-conventions/issues/226>
+
+### done
+
+- ([@frzifus](https://github.com/frzifus)) <https://github.com/open-telemetry/semantic-conventions/issues/98>
+
+#### Optional
+
+- <https://github.com/open-telemetry/semantic-conventions/issues/648>
+- <https://github.com/open-telemetry/semantic-conventions/issues/531>
+
+### Finally
+
+- Stabilize existing semantic conventions (system metrics)
+
+## Deliverables
+
+- Mark the [system semantic conventions](https://github.com/open-telemetry/opentelemetry-specification/blob/37ab6b056172ad36d31ce217927e47bbe031d831/specification/metrics/semantic_conventions/system-metrics.md#semantic-conventions-for-system-metrics) as stable.
+- Update existing OpenTelemetry Collector hostmetricsreceiver to conform with the stable conventions.
+
+## Staffing / Help Wanted
+
+The goal is to follow [@tedsuo](http://github.com/tedsuo)'s proposed [Semantic Convention Process](https://docs.google.com/document/d/1ghvajKaipiNZso3fDtyNxU7x1zx0_Eyd02OGpMGEpLE/edit#heading=h.xc2ft2cddhny).
+
+- **Stage 1**: SIG Preparation
+- **Stage 2**: Stabilizing the Specification
+- **Stage 3**: Implementation
+
+### Required staffing
+
+- [@mx-psi](https://github.com/mx-psi), [@frzifus](https://github.com/frzifus) project leads
+- [@jsuereth](https://github.com/jsuereth) technical committee sponsor
+- domain experts:
+  - [@MovieStoreGuy](https://github.com/MovieStoreGuy)
+  - [@dmitryax](https://github.com/dmitryax)
+  - [@gbbr](https://github.com/gbbr)
+  - [@sallyom](https://github.com/sallyom)
+  - [@ChrsMark](https://github.com/ChrsMark)
+  - [@kaiyan-sheng](https://github.com/kaiyan-sheng)
+  - [@braydonk](https://github.com/braydonk)
+  - [@bertysentry](https://github.com/bertysentry)
+  - [@dineshg13](https://github.com/dineshg13)
+
+## Meeting Times
+
+July 10 - August 21 (6 weeks):
+
+- Wednesday at 8:30-09:00 Pacific Time
```

Comment:
I know this was the original time. However could this be misleading since we have changed the time since then? Maybe an "update" section bellow could help here?

Reply:
I think that's a valid concern, but this applies to other documents as well, maybe something we fix for all of them at once.

Reply:
I think it would also be ok to remove the meeting times from this document

Reply:
good idea, I removed the meeting times.